Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -86,69 +152,46 @@ class AppSettingsTableViewController: SettingsTableViewController {
         // There is nothing to show in the Customize section if we don't include the compact tab layout
         // setting on iPad. When more options are added that work on both device types, this logic can
         // be changed.
-
         generalSettings += [
             BoolSetting(prefs: prefs, prefKey: "showClipboardBar", defaultValue: false,
                         titleText: Strings.Toast.SettingsOfferClipboardBarTitle,
                         statusText: Strings.Toast.SettingsOfferClipboardBarStatus),
         ]
-        settings += [ SettingSection(title: NSAttributedString(string: Strings.Settings.General.SectionTitle), children: generalSettings)]
-
-        settings += [
-            SettingSection(title: NSAttributedString(string: NSLocalizedString("Support", comment: "Support section title")), children: [
-                ShowIntroductionSetting(settings: self),
-                SendFeedbackSetting(),
-                BoolSetting(prefs: prefs, prefKey: AppConstants.PrefSendUsageData, defaultValue: true, attributedTitleText: NSAttributedString(string: Strings.Settings.SendUsage.Title), attributedStatusText: NSAttributedString(string: Strings.Settings.SendUsage.Message, attributes: [NSAttributedString.Key.foregroundColor: Theme.tableView.headerTextLight])),
-            ]),
-            SettingSection(title: NSAttributedString(string: NSLocalizedString("About", comment: "About settings section title")), children: [
-                VersionSetting(settings: self),
-                LicenseAndAcknowledgementsSetting(),
-                ExportBrowserDataSetting(settings: self),
-                ExportLogDataSetting(settings: self),
-                DeleteExportedDataSetting(settings: self),
-                ForceCrashSetting(settings: self),
-                SlowTheDatabase(settings: self),
-                SentryIDSetting(settings: self),
-            ]), ]
-
-        return settings
+        return SettingSection(title: NSAttributedString(string: Strings.Settings.General.SectionTitle), children: generalSettings)
     }
 
-    override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let headerView = super.tableView(tableView, viewForHeaderInSection: section) as! ThemedTableSectionHeaderFooterView
-        // Prevent the top border from showing for the General section.
-        switch section {
-        case 1:
-            headerView.showTopBorder = false
-        default:
-            break
-        }
-
-        return headerView
+    private func newsSettingSection() -> SettingSection {
+        let prefs = self.profile.prefs
+        let newsSettigns = [
+            NewsLanguageSetting(currentRegion: self.newsCurrentRegion, availableRegions: self.newsAvailableRegions),
+            BoolSetting(prefs: prefs, prefKey: PrefsKeys.NewsFromNewTabEnabled, defaultValue: true, titleText: Strings.Settings.News.NewsFromNewTabPage),
```

Comment:
How about we call those prefs: `NewTabNewsEnabled` and `NewTabNewsImagesEnabled`?